pkg/distributor/push.go

```diff
@@ -177,7 +177,11 @@ func handler(
 				msg = err.Error()
 			}
 			if code != 202 {
-				level.Error(logger).Log("msg", "push error", "err", err)
+				msgs := []interface{}{"msg", "push error", "httpCode", code, "err", err}
```

Comment:
[non blocking]  if we change the `msg` here to be slightly more descriptive, given we'll show it to customer too?

_Same comment applies below._

Reply:
👍 In our insight dashboard, we only show `err` part to the user, but user can find entire message.

Reply:
Changed to `failed to ingest data from incoming request`, happy to use better alternative.

Reply:
Ended with:
* `detected an error while ingesting Prometheus remote-write request (the request may have been partially ingested)`
* `detected an error while ingesting OTLP metrics request (the request may have been partially ingested)`

Which is similar to ingester's message in ingest-storage mode when ingestion fails due to client-error:
* `detected a client error while ingesting write request (the request may have been partially ingested)`